tools/src/test/java/org/apache/kafka/tools/reassign/ReassignPartitionsUnitTest.java

```diff
@@ -0,0 +1,788 @@
+                alterPartitionReassignments(adminClient, asScala(reassignments));
+
+            assertTrue(reassignmentResult.isEmpty());
+            assertEquals(String.join(System.lineSeparator(),
+                "Current partition reassignments:",
+                "bar-0: replicas: 2,3,0. removing: 0.",
+                "foo-0: replicas: 0,1,2. adding: 4.",
+                "foo-1: replicas: 1,2,3. adding: 4,5. removing: 1,2."),
+                curReassignmentsToString(adminClient));
+        }
+    }
+
+    private void verifyBrokerThrottleResults(Config config,
+                                             long expectedInterBrokerThrottle,
+                                             long expectedReplicaAlterLogDirsThrottle) {
+        Map<String, String> configs = new HashMap<>();
+        config.entries().forEach(entry -> configs.put(entry.name(), entry.value()));
+        if (expectedInterBrokerThrottle >= 0) {
+            assertEquals(Long.toString(expectedInterBrokerThrottle),
+                configs.getOrDefault(brokerLevelLeaderThrottle(), ""));
+            assertEquals(Long.toString(expectedInterBrokerThrottle),
+                configs.getOrDefault(brokerLevelFollowerThrottle(), ""));
+        }
+        if (expectedReplicaAlterLogDirsThrottle >= 0) {
+            assertEquals(Long.toString(expectedReplicaAlterLogDirsThrottle),
+                configs.getOrDefault(brokerLevelLogDirThrottle(), ""));
+        }
+    }
+
+    @Test
+    public void testModifyTopicThrottles() throws Exception {
+        try (MockAdminClient adminClient = new MockAdminClient.Builder().numBrokers(4).build()) {
+            addTopics(adminClient);
+
+            Map<String, String> leaderThrottles = new HashMap<>();
+
+            leaderThrottles.put("foo", "leaderFoo");
+            leaderThrottles.put("bar", "leaderBar");
+
+            modifyTopicThrottles(adminClient,
+                asScala(leaderThrottles),
+                asScala(Collections.singletonMap("bar", "followerBar")));
+            List<ConfigResource> topics = Stream.of("bar", "foo").map(
+                id -> new ConfigResource(ConfigResource.Type.TOPIC, id)).collect(Collectors.toList());
+            Map<ConfigResource, Config> results = adminClient.describeConfigs(topics).all().get();
+            verifyTopicThrottleResults(results.get(topics.get(0)), "leaderBar", "followerBar");
+            verifyTopicThrottleResults(results.get(topics.get(1)), "leaderFoo", "");
+        }
+    }
+
+    private void verifyTopicThrottleResults(Config config,
+                                            String expectedLeaderThrottle,
+                                            String expectedFollowerThrottle) {
+        Map<String, String> configs = new HashMap<>();
+        config.entries().forEach(entry -> configs.put(entry.name(), entry.value()));
+        assertEquals(expectedLeaderThrottle,
+            configs.getOrDefault(ReassignPartitionsCommand.topicLevelLeaderThrottle(), ""));
```

Comment:
Why not static import for  `ReassignPartitionsCommand#topicLevelLeaderThrottle` like for other members of the  `ReassignPartitionsCommand`?

Reply:
Fixed. Thanks.